features/backup/src/main/java/ir/mehdiyari/krypt/backup/DataRoute.kt

```diff
@@ -26,7 +35,34 @@ internal fun DataRoute(
     modifier: Modifier,
 ) {
     val managerStoragePermissionState = remember { mutableStateOf(false) }
-    DataScreenScaffold(modifier = modifier, onNavigationClicked = onNavigationClicked) {
+
+    val chooseDirectorySnackbarMsg =
+        stringResource(id = R.string.choose_backup_directory_description)
+    val context = LocalContext.current
+
+    var selectedDirectory: DocumentFile? by remember { mutableStateOf(null) }
+    val launcher = rememberLauncherForActivityResult(
```

Comment:
Would you please rename this variable to something more meaningful based on the job of the variable?